src/components/v5/common/AvatarUploader/partials/DefaultContent.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import Icon from '~shared/Icon';
+import styles from './AvatarUploader.module.css';
+import { DefaultContentProps } from '../types';
+import { useMobile } from '~hooks';
+
+const displayName = 'v5.common.AvatarUploader.partials.SuccessContent';
+
+const DefaultContent: FC<DefaultContentProps> = ({ open }) => {
+  const { formatMessage } = useIntl();
+  const isMobile = useMobile();
+
+  return (
+    <div
+      className={clsx(
+        'flex-col items-center bg-white-100 border-gray-200',
+        styles.contentWrapper,
+      )}
+    >
+      <div className="w-[2.55rem] mb-[0.5rem]">
```

Comment:
Please, correct all custom sizes in `[]` in this file.
There are incorrect.
https://tailwindcss.com/docs/customizing-spacing